.eslintrc

```diff
@@ -1,5 +1,8 @@
 {
     "parser": "babel-eslint",
+    "extends": [
+      "defaults/configurations/google"
```

Comment:
why the google config over the default eslint config (https://github.com/eslint/eslint/blob/master/conf/eslint.json) ? Also I'm not especially convinced to use a default config, do you think it's better than using our own ?